sim/rogue/dps_rogue/dps_rogue_test.go

```diff
@@ -49,7 +49,7 @@ func TestCombat(t *testing.T) {
 		},
 		{
 			Class:      proto.Class_ClassRogue,
-			Phase:		5,
+			Phase:      5,
```

Comment:
I'm not sure what changed in these test files or the bash script below but lucenia will probably flag these for removal

Reply:
TBH I have no idea how this got here... I haven't intentionally even looked at these files in my editor.
Will revert.